LaTeX/LaTeX.sublime-syntax

```diff
@@ -1655,3 +1657,30 @@ contexts:
        3: punctuation.definition.group.brace.begin.latex
        4: entity.name.function.frame.latex
        5: punctuation.definition.group.brace.end.latex
+
+  array-preamble:
+    - match: '(>)\s*(\{)'
```

Comment:
As far I know this is only possible inside column definitions, i.e. `\newcolumntype{X}{>{...}l}` and `begin{tabular}{|l|>{...}l}`, so we should not match this everywhere.